src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -740,13 +752,13 @@ case class PUnnamedReceiver(typ: PMethodRecvType) extends PReceiver
 case class PResult(outs: Vector[PParameter]) extends PNode with PActualMisc
 
 sealed trait PEmbeddedType extends PNode with PActualMisc {
-  def typ: PNamedType
+  def typ: PUnqualifiedTypeName
   def name: String = typ.name
 }
 
-case class PEmbeddedName(typ: PNamedType) extends PEmbeddedType
+case class PEmbeddedName(typ: PUnqualifiedTypeName) extends PEmbeddedType
```

Comment:
I know we discussed this already. But could you please remind me what the difference between PTypeName and PUnqualfiedTypeName is?

Reply:
PTypeName corresponds to [TypeName](https://go.dev/ref/spec#TypeName) in the go specification, while PUnqualifiedTypeName corresponds to types that are just identifiers without a dot, including predeclared types.